package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "url-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "url-shortner.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
nit: clean up unused parts of the package.json. You probably don't need the description, main, or scripts section here.